[2024-07-03T09:57:54.089+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T12:50:00+00:00 [queued]>
[2024-07-03T09:57:54.110+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T12:50:00+00:00 [queued]>
[2024-07-03T09:57:54.111+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-07-03T09:57:54.141+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): id_task_get_emails> on 2024-07-02 12:50:00+00:00
[2024-07-03T09:57:54.148+0000] {standard_task_runner.py:57} INFO - Started process 20649 to run task
[2024-07-03T09:57:54.160+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'reports_workflow', 'id_task_get_emails', 'scheduled__2024-07-02T12:50:00+00:00', '--job-id', '399', '--raw', '--subdir', 'DAGS_FOLDER/dag_email_robot.py', '--cfg-path', '/tmp/tmpg7kyu6wd']
[2024-07-03T09:57:54.161+0000] {standard_task_runner.py:85} INFO - Job 399: Subtask id_task_get_emails
[2024-07-03T09:57:54.252+0000] {task_command.py:415} INFO - Running <TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T12:50:00+00:00 [running]> on host 6f4ef5488251
[2024-07-03T09:57:54.370+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='FIRDAVS' AIRFLOW_CTX_DAG_ID='reports_workflow' AIRFLOW_CTX_TASK_ID='id_task_get_emails' AIRFLOW_CTX_EXECUTION_DATE='2024-07-02T12:50:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-02T12:50:00+00:00'
[2024-07-03T09:57:54.371+0000] {logging_mixin.py:151} INFO - path of config file:  /opt/***
[2024-07-03T09:57:54.371+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/dag_email_robot.py", line 22, in get_emails
    email_conf = json.load (open(path / ("config/email_conf.json")))
NameError: name 'json' is not defined
[2024-07-03T09:57:54.390+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=reports_workflow, task_id=id_task_get_emails, execution_date=20240702T125000, start_date=20240703T095754, end_date=20240703T095754
[2024-07-03T09:57:54.414+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 399 for task id_task_get_emails (name 'json' is not defined; 20649)
[2024-07-03T09:57:54.450+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-07-03T09:57:54.482+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-03T12:05:21.838+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T12:50:00+00:00 [queued]>
[2024-07-03T12:05:21.850+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T12:50:00+00:00 [queued]>
[2024-07-03T12:05:21.850+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-07-03T12:05:21.869+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): id_task_get_emails> on 2024-07-02 12:50:00+00:00
[2024-07-03T12:05:21.874+0000] {standard_task_runner.py:57} INFO - Started process 23460 to run task
[2024-07-03T12:05:21.881+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'reports_workflow', 'id_task_get_emails', 'scheduled__2024-07-02T12:50:00+00:00', '--job-id', '840', '--raw', '--subdir', 'DAGS_FOLDER/dag_email_robot.py', '--cfg-path', '/tmp/tmpa958ylm0']
[2024-07-03T12:05:21.882+0000] {standard_task_runner.py:85} INFO - Job 840: Subtask id_task_get_emails
[2024-07-03T12:05:21.946+0000] {task_command.py:415} INFO - Running <TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T12:50:00+00:00 [running]> on host 6f4ef5488251
[2024-07-03T12:05:22.092+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='FIRDAVS' AIRFLOW_CTX_DAG_ID='reports_workflow' AIRFLOW_CTX_TASK_ID='id_task_get_emails' AIRFLOW_CTX_EXECUTION_DATE='2024-07-02T12:50:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-02T12:50:00+00:00'
[2024-07-03T12:05:22.093+0000] {logging_mixin.py:151} INFO - path of config file:  /opt/***
[2024-07-03T12:05:22.093+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:05:22.093904_User@StatDep: EmailRobot in screated.
[2024-07-03T12:05:22.094+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:05:22.094118_User@StatDep: get_for_recent_emails is started...
[2024-07-03T12:05:22.095+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:05:22.095480_User@StatDep: pop3_fetch_emails isstarted...
[2024-07-03T12:05:22.476+0000] {logging_mixin.py:151} INFO -    b'+OK'
[2024-07-03T12:05:22.892+0000] {logging_mixin.py:151} INFO -    mailbox have 1 emails and the robot can process 10
[2024-07-03T12:05:23.375+0000] {logging_mixin.py:151} INFO -    #>2024-07-03 12:05:23.375481_conneciton to db is set.
[2024-07-03T12:05:23.383+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/dag_email_robot.py", line 25, in get_emails
    robot.fetch_emails()
  File "/opt/airflow/dags/emailRobot.py", line 62, in fetch_emails
    return self.pop3_fetch_emails()
  File "/opt/airflow/dags/emailRobot.py", line 156, in pop3_fetch_emails
    cursor.execute (query, insert_values)
psycopg2.errors.UndefinedTable: relation "sma_stat_dep.tbl_file_upload" does not exist
LINE 1: ...) AS (INSERT INTO                                 sma_stat_d...
                                                             ^

[2024-07-03T12:05:23.417+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=reports_workflow, task_id=id_task_get_emails, execution_date=20240702T125000, start_date=20240703T120521, end_date=20240703T120523
[2024-07-03T12:05:23.496+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 840 for task id_task_get_emails (relation "sma_stat_dep.tbl_file_upload" does not exist
LINE 1: ...) AS (INSERT INTO                                 sma_stat_d...
                                                             ^
; 23460)
[2024-07-03T12:05:23.541+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-07-03T12:05:23.655+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-03T12:24:38.480+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T12:50:00+00:00 [queued]>
[2024-07-03T12:24:38.487+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T12:50:00+00:00 [queued]>
[2024-07-03T12:24:38.488+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-07-03T12:24:38.519+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): id_task_get_emails> on 2024-07-02 12:50:00+00:00
[2024-07-03T12:24:38.524+0000] {standard_task_runner.py:57} INFO - Started process 24442 to run task
[2024-07-03T12:24:38.534+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'reports_workflow', 'id_task_get_emails', 'scheduled__2024-07-02T12:50:00+00:00', '--job-id', '1187', '--raw', '--subdir', 'DAGS_FOLDER/dag_email_robot.py', '--cfg-path', '/tmp/tmp2n2kj2_i']
[2024-07-03T12:24:38.535+0000] {standard_task_runner.py:85} INFO - Job 1187: Subtask id_task_get_emails
[2024-07-03T12:24:38.643+0000] {task_command.py:415} INFO - Running <TaskInstance: reports_workflow.id_task_get_emails scheduled__2024-07-02T12:50:00+00:00 [running]> on host 6f4ef5488251
[2024-07-03T12:24:38.781+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='FIRDAVS' AIRFLOW_CTX_DAG_ID='reports_workflow' AIRFLOW_CTX_TASK_ID='id_task_get_emails' AIRFLOW_CTX_EXECUTION_DATE='2024-07-02T12:50:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-02T12:50:00+00:00'
[2024-07-03T12:24:38.782+0000] {logging_mixin.py:151} INFO - path of config file:  /opt/***
[2024-07-03T12:24:38.782+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:24:38.782860_User@StatDep: EmailRobot in screated.
[2024-07-03T12:24:38.783+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:24:38.783033_User@StatDep: get_for_recent_emails is started...
[2024-07-03T12:24:38.783+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:24:38.783164_User@StatDep: pop3_fetch_emails isstarted...
[2024-07-03T12:24:39.189+0000] {logging_mixin.py:151} INFO -    b'+OK'
[2024-07-03T12:24:39.474+0000] {logging_mixin.py:151} INFO -    mailbox have 0 emails and the robot can process 10
[2024-07-03T12:24:39.474+0000] {logging_mixin.py:151} INFO - #>2024-07-03 12:24:39.474710_User@StatDep: no emails to process, exiting pop3_fetch_emails()...
[2024-07-03T12:24:39.475+0000] {python.py:194} INFO - Done. Returned value was: True
[2024-07-03T12:24:39.594+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=reports_workflow, task_id=id_task_get_emails, execution_date=20240702T125000, start_date=20240703T122438, end_date=20240703T122439
[2024-07-03T12:24:39.661+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-07-03T12:24:39.699+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
